integration-tests/bats/sql-merge.bats

```diff
@@ -1669,9 +1669,9 @@ SQL
     dolt commit -am "changes to b2"
 
     dolt checkout main
-    run dolt merge b1
+    run dolt merge b1 -m "merge b1"
```

Comment:
Would be good to throw in the `--commit` arg somewhere, like here. Just to make sure we have a test that uses it, even though it's the default. 

Speaking of... do we have code to error out if `--commit` and `--no-commit` are both specified? That would be a good error case to test, too.